index.js

```diff
@@ -15,3 +16,171 @@ navMenu.addEventListener('click', () => {
+        <li>${cards.technologies[0]}</li>
+        <li>${cards.technologies[1]}</li>
+        <li>${cards.technologies[2]}</li>
+      </ul>
+      </div>
+      <a href="#" class="btn_2 ${btnClass} popup1-btn">See Project</a>
+      `;
+  }
+  for (let i = 1; i <= 6; i += 1) {
+    if (i === 1) {
+      const layout = document.createElement('div');
+      const firstCard = document.querySelector(`.card${i}`);
+      layout.innerHTML = cardInstancefunc('nonvisible', '', '', '');
+      firstCard.appendChild(layout);
+    } else if (i === 2 || i === 5) {
+      const layout = document.createElement('div');
+      const secondCard = document.querySelector(`.card${i}`);
+      layout.innerHTML = cardInstancefunc('', 'nodisplay-desktop', '<h2 class="hidden-mobile">Data Dashboard Healthcare</h2>', 'nodisplay-desktop');
+      secondCard.appendChild(layout);
+    } else if (i === 3 || i === 6) {
+      const layout = document.createElement('div');
+      const thirdCard = document.querySelector(`.card${i}`);
+      layout.innerHTML = cardInstancefunc('', 'nodisplay-desktop', '<h2 class="hidden-mobile">Website Portfolio</h2>', 'nodisplay-desktop');
+      thirdCard.appendChild(layout);
+    } else {
+      const layout = document.createElement('div');
+      const fourthCard = document.querySelector(`.card${i}`);
+      layout.innerHTML = cardInstancefunc('', 'nodisplay-desktop', '', '');
+      fourthCard.appendChild(layout);
+    }
+  }
+});
+
+const modalContent = [
+  {
+    projectName: 'Multi Post Stories',
+    featuredImageDesk: '<img class="hidden-mobile modal-desktop" src="images/modal-desktop.png" alt="portfolio snapshot">',
+    featureImageMobile: '<img class="nodisplay-desktop modal-mobile" src="images/modal-protfolio.png" alt="portfolio snapshot">',
+    closeBtn: 'images/modal-close.png',
+    featureDescriptionMob: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    featureDescriptionDesk: `Lorem Ipsum is simply dummy text of the printing and typesetting
+    industry. Lorem Ipsum
+    has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of
+    type
+    and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting
+    industry.
+    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+    galley of type and scrambled it 1960s with the releax map lapora verita.`,
+    technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+    liveLink: 'https://wale-prog.github.io/My-porfolio',
+    sourceLink: 'https://github.com/wale-prog/My-porfolio',
+  },
+];
```

Comment:
kindly this window should contain `see live button` and `source code with a github logo` on the mobile device. add them to all windows for each project

Reply:
Thank you for the feedback, I don't know how I missed that in the CSS file but it has been fixed now. Thanks.